sdk/purview/azure-purview-sharing/samples/received_shares_examples.py

```diff
@@ -0,0 +1,121 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+# These examples are ingested by the documentation system, and are
+# displayed in the SDK reference documentation. When editing these
+# example snippets, take into consideration how this might affect
+# the readability and usability of the reference documentation.
+
+# Create a received share client
+
+# [START create_a_received_share_client]
+import os
+
+from azure.purview.sharing import PurviewSharing
+from azure.identity import ClientSecretCredential
+
+endpoint = os.environ["ENDPOINT"]
+tenant_id = os.environ.get("AZURE_TENANT_ID", getattr(os.environ, "TENANT_ID", None))
+client_id = os.environ.get("AZURE_CLIENT_ID", getattr(os.environ, "CLIENT_ID", None))
+secret = os.environ.get("AZURE_CLIENT_SECRET", getattr(os.environ, "CLIENT_SECRET", None))
+credential = ClientSecretCredential(tenant_id=str(tenant_id), client_id=str(client_id), client_secret=str(secret))
```

Comment:
can you switch this to DefaultAzureCredential? It will clean up the code for environment variables, and it seems that's more of a hero scenario